### PR TITLE
better naming and description of textindexer in behaviors control panel

### PR DESCRIPTION
**Pull request by jensens**
less clumpsy, easier to get,

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: plone/app/dexterity/textindexer/configure.zcml [hunk 1]
```diff
@@ -6,9 +6,9 @@
 
     <!-- marker behavior for enabling indexer -->
     <plone:behavior
-          title="Dynamic SearchableText indexer behavior"
+          title="Full-Text Indexing"
           name="plone.textindexer"
-          description="Enables the dynamic SearchableText indexer for a content type"
+          description="Enables the enhanced full-text indexing for a content type ('plone.textindexer'). If a field is marked 'searchable', it's content got added to the 'SearchableText' index in the catalog."
```
Comment (mauritsvanrees) [inline plone/app/dexterity/textindexer/configure.zcml:11]: * `it's` -> `its`
* `got` -> `gets`

**Review by mauritsvanrees — APPROVED** [review]
LGTM except a comment about typos.
Build is green, can be merged when fixed (or fixed after merge).

File: plone/app/dexterity/textindexer/configure.zcml (hunk 1 above)
Comment (jensens) [reply plone/app/dexterity/textindexer/configure.zcml:11]:
```suggestion
          description="Enables the enhanced full-text indexing for a content type ('plone.textindexer'). If a field is marked 'searchable', its content gets added to the 'SearchableText' index in the catalog."
```

**Pull request merged**